qiskit_ibm_runtime/runtime_job.py

```diff
@@ -394,6 +395,11 @@ def metrics(self) -> Dict[str, Any]:
             IBMRuntimeError: If a network error occurred.
         """
         try:
+            issue_deprecation_msg(
+                msg="The 'bss.seconds' attribute is deprecated",
+                version="0.11",
+                remedy="Use 'usage.quantum_seconds' attribute instead.",
```

Comment:
```suggestion
                version="0.11.1",
                remedy="Use the 'usage.quantum_seconds' attribute instead.",
```

Small comment but otherwise looks good - we can have this in the next release in 2 weeks 0.11.1